SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2815,17 +2800,50 @@ bool USpatialNetDriver::HasTimedOut(const float Interval, uint64& TimeStamp)
 }
 
 // This should only be called once on each client, in the SpatialDebugger constructor after the class is replicated to each client.
-void USpatialNetDriver::SetSpatialDebugger(ASpatialDebugger* InSpatialDebugger)
+void USpatialNetDriver::RegisterSpatialDebugger(ASpatialDebugger* InSpatialDebugger)
 {
-	check(!IsServer());
-	if (SpatialDebugger != nullptr)
+	if (!SpatialDebuggerSystem.IsValid())
 	{
-		UE_LOG(LogSpatialOSNetDriver, Error, TEXT("SpatialDebugger should only be set once on each client!"));
-		return;
+		typedef SpatialGDK::FSubView FSubView;
```

Comment:
nit: these typdefs are a little unorthodox for gdk code, we normally as using statements

Reply:
Yeah I tried that - would mix SpatialDebuggerSystem as a field name and as a class name. I can go for

```
using namespace SpatialGDK;

this->SpaitalDebuggerSystem = MakeUnique<SpatialGDK::SpatialDebuggerSystem>(...);
```

Reply:
using namespace SpatialGDK::FSubView etc. ?